lightning/src/ln/channel.rs

```diff
@@ -4132,6 +4148,18 @@ impl<Signer: Sign> Channel<Signer> {
 					data: format!("Funding transaction was un-confirmed. Locked at {} confs, now have {} confs.", self.minimum_depth.unwrap(), funding_tx_confirmations),
 				});
 			}
+		} else if let Some(creation_height) = self.channel_creation_height {
+			if !self.is_outbound() && self.funding_tx_confirmed_in.is_none() &&
```

Comment:
would it make sense to close pending outbound channels after a while, too?

Reply:
We cannot - we've broadcasted the funding transaction and have to wait for it to confirm. Of course we could go ahead and either double spend it or broadcast the initial commitment transaction, but the first has to be done by the LDK user (though maybe we could make it more clear that they're allowed to), but the second could also be done with a call to force_close_channel.